### DISCUSSION TITLE
Contact

### DISCUSSION THREAD

**Discussion by SensorsIot** [erstec/APRS-ESP#45] [General]
Hi Ernest,
I would be glad if you contact me to discuss your project. We work on a similar one but with an I2S audio interface. My email is: sensorsiot@gmail.com
Regards
Andreas, HB9BLA

**Comment by erstec** [comment]
Hi Andreas!
Please write me on ly3ph@qsl.lt or volwerene on Discord

73, LY3PH Ernest
